### PR TITLE
Add tutorial for EventLogger plugin

### PR DESCRIPTION
**Pull request by sbis04**
I have added the tutorial for the `EventLogger` plugin. The page contents are as follows:

> The updated screenshot of this page is in a comment below.

EventLogger plugin

Tag #891

### CODE REVIEW

File: docusaurus/docs/using-plugin-with-wechaty/event-logger.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+
+Before proceeding further, you should install the dependencies that we just specified in the `package.json` file. Run the following command from the root directory:
+
+```sh
+npm install
+```
+
+This will generate a `node_modules` directory containing all the installed dependencies, and a `package-lock.json` file.
+
+### 4. Writing code for bot
```
Comment (proudofsimin) [inline docusaurus/docs/using-plugin-with-wechaty/event-logger.md]: Write

File: docusaurus/docs/using-plugin-with-wechaty/event-logger.md [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+// Initializing the bot
+const bot = new Wechaty({
+    name: 'event-logger-bot',
+})
+
+// Starting the bot
+bot.start()
+```
+
+### 5. Running the bot
```
Comment (proudofsimin) [inline docusaurus/docs/using-plugin-with-wechaty/event-logger.md]: Run

File: docusaurus/docs/using-plugin-with-wechaty/event-logger.md [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+
+### 1. Add dependency
+
+As the `EventLogger` plugin is present in the `wechaty-plugin-contrib` NPM package, you have to first add it to the dependencies. It can be installed using the following command:
+
+```sh
+npm i wechaty-plugin-contrib
+```
+
+### 2. Integrating the plugin
```
Comment (proudofsimin) [inline docusaurus/docs/using-plugin-with-wechaty/event-logger.md]: Integrate

**Review by proudofsimin — APPROVED** [review]
Hi, 
This is great. Please just change the v-ing in steps to present tense. Thank you.

**Comment by sbis04** [comment]
@proudofsimin I have changed the -ing(s) in steps to present tense and fixed the title capitalizations.

**Comment by sbis04** [comment]
Hi @proudofsimin @Rohitesh-Kumar-Jain in the previous meeting we were confused where to keep the simple bot, so that we can use it to show how to add the plugins.

I and @shwetalsoni discussed about this, and we decided to keep the simple bot inside **Examples > Basic > Starter Bot**. And, we will show how to add the plugins using this bot.

Sidebar examples basic

The updated **Event Logger plugin** page looks like this:

Event Logger plugin

File: docusaurus/docs/using-plugin-with-wechaty/event-logger.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+* [macOS](https://nodejs.org/en/download/package-manager/#macos)
+
+> Installation guide for `Node.js` on other platforms can be found [here](https://nodejs.org/en/download/package-manager/).
+
+:::
+
+## Adding Event Logger plugin
+
+For the demonstration of adding this plugin, we will use the **Starter Bot** and show you how to add the `EventLogger` plugin to it. Follow the steps below:
+
```
Comment (proudofsimin) [inline docusaurus/docs/using-plugin-with-wechaty/event-logger.md]: Hi, do you think if it's better if we add a step one telling our user to create a simple bot first? Anyway it is still a step they need to do first.

1. Create a starter bot

Follow the instructions in [Starter bot] to create the the foundation.

2. Install dependency
...

**Review by proudofsimin — COMMENTED** [review]
I love your new organization so much. This is so good!

File: docusaurus/docs/using-plugin-with-wechaty/event-logger.md (hunk 4 above)
Comment (sbis04) [reply docusaurus/docs/using-plugin-with-wechaty/event-logger.md]: Yes, I have added step one for telling the users to create a starter bot

File: docusaurus/docs/using-plugin-with-wechaty/event-logger.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+### 1. Create a starter bot
+
+Follow the instructions on the **Starter Bot** page to create the foundation of a Wechaty bot.
```
Comment (lijiarui) [inline docusaurus/docs/using-plugin-with-wechaty/event-logger.md]: I thinks we should add link for `**Starter Bot*` page and then users can refer that part if he don't know how to build a starter bot.

File: docusaurus/docs/using-plugin-with-wechaty/event-logger.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+// Add the following line to use the plugin
+bot.use(EventLogger())
+bot.start()
+```
+
+### 4. Run the bot
+
+In order to run the bot, first you have to **export/set** an environment variable with the type of puppet to use, and then start the bot:
+
+#### Linux/macOS
```
Comment (lijiarui) [inline docusaurus/docs/using-plugin-with-wechaty/event-logger.md]: I think we should unify the style here, @shwetalsoni did a good job, it looks like this：

image

You can use code like this:

```markdown
<Tabs
  groupId="operating-systems"
  defaultValue="linux"
  values={[
    { label: 'Linux',   value: 'linux', },
    { label: 'macOS',   value: 'mac', },
    { label: 'Windows', value: 'windows', },
  ]
}>

<TabItem value="linux">

```

File: docusaurus/docs/using-plugin-with-wechaty/event-logger.md [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+Congratulations! You have successfully integrated the `EventLogger` plugin to your Wechaty bot. You will see after running the bot, it starts logging the Wechaty events.
+
+## Conclusion
+
+You can apply a similar concept to add the `EventLogger` plugin to any of your Wechaty bots. You can learn more about this plugin [here](https://github.com/wechaty/wechaty-plugin-contrib#2-eventlogger).
+
+Learn how to integrate [QR Code Terminal Plugin](./qr-code-terminal) to your Wechaty bot in the next tutorial.
+
+## References
+
```
Comment (lijiarui) [inline docusaurus/docs/using-plugin-with-wechaty/event-logger.md]: Add video link reference here: https://www.youtube.com/watch?v=tfGZXoe_aA4

**Review by lijiarui — CHANGES_REQUESTED** [review]
Pls change as I request, thanks!

File: docusaurus/docs/using-plugin-with-wechaty/event-logger.md (hunk 5 above)
Comment (sbis04) [reply docusaurus/docs/using-plugin-with-wechaty/event-logger.md]: Yes, I'll add the link. Currently, the **Starter Bot** page isn't there on the docs, we have it further down on our proposal timeline. Once, the page is ready we will also update the link here.

**Comment by lijiarui** [comment]
ping @sbis04

**Comment by lijiarui** [comment]
@sbis04 Pls resolve the conflict, then I will approve this PR.

**Comment by sbis04** [comment]
@lijiarui Have made the CI green.

**Review by lijiarui — APPROVED** [review]
LGTM

**Comment by lijiarui** [comment]
@sbis04  Since the `.webp` format has lots of advantages and the [safari has supported it last year](https://www.macrumors.com/2020/06/22/webp-safari-14/) which means all the modern browsers will be able to compatible with it.

@huan has just added a new unit test for enforcing the image format: the CI will not be passed if we are using other image formats than the perfect .webp.

Please make CI green to adjust the new rule, see https://github.com/wechaty/wechaty.js.org/issues/1035

**Comment by sbis04** [comment]
@lijiarui I have converted the image to `.webp` format and made the CI green.

**Review by huan — APPROVED** [review]

**Pull request merged**